.gitignore

```diff
@@ -1,6 +1,6 @@
 composer.phar
 vendor/
-
+.idea*
```

Comment:
I'm not being critical at all, i'm simply offering an point of view that is commonly accepted by the open source community.